### ISSUE TITLE
Syntax error using readthedocs

### ISSUE THREAD

**Issue by brieang** [mkdocstrings/mkdocstrings#322]
Hello all,

Using the mkdocs library to build my documentation I have the following error, since the last mkdocstrings update in v0.16.0:
```
  File "/usr/local/lib/python3.8/site-packages/mkdocstrings/templates/python/readthedocs/return.html", line 12, in template
    <li>{{ (("`" + return.annotation + "` – ") if return.annotation else "") + return.description)|convert_markdown(heading_level, html_id) }}</li>
jinja2.exceptions.TemplateSyntaxError: unexpected ')'
```

Looking at the mkdocstrings code, it seems indeed this line has more closed parenthesis than opened ones:
https://github.com/mkdocstrings/mkdocstrings/blob/b05569c292b08df4b6f363966f80dec17ada8aa5/src/mkdocstrings/templates/python/readthedocs/return.html#L12

It doesn't seems to have been changed recently, so I don't know why it fails only since I updated mkdocstrings in v0.16.0 ...

**Comment by pawamoy** [comment]
Hello @brieang, thank you for the report and sorry for the breakage! There's indeed an extra closing parenthesis here. I'll push a fix right now. If you share your repo, we can add it to our list of repos tested for regressions (this would be useful to have one using the readthedocs theme).

**Comment by pawamoy** [comment]
Obviously we should also add tests for the themes we claim to support :smile:

**Comment by brieang** [comment]
Thanks, for the quick answer !
Unfortunately I can't share the repo I am working on, because it is not a Github one, that is a private company repo ;)

**Comment by pawamoy** [comment]
Fixed in 0.16.1 :slightly_smiling_face:
